src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -16,6 +16,7 @@
     <!-- Disable native AOT on FreeBSD when cross building from Linux. -->
     <NativeAotSupported Condition="'$(TargetOS)' == 'freebsd' and '$(CrossBuild)' == 'true'">false</NativeAotSupported>
     <PublishAot Condition="'$(NativeAotSupported)' == 'true'">true</PublishAot>
+    <StripSymbols Condition="'$(DotNetBuildSourceOnly)' == 'true' and '$(KeepNativeSymbols)' == 'true'">false</StripSymbols>
```

Comment:
`KeepNativeSymbols` is initialized as

https://github.com/dotnet/runtime/blob/ae6ad886bb37297912689a4edd4d29988585d29a/Directory.Build.props#L393-L394

We _could_ leave out the `DotNetBuildSourceOnly` condition here, if desired.

Reply:
@MichalStrehovsky do you want to keep this as is, or remove the `DotNetBuildSourceOnly` condition?

Reply:
Dropping the `'$(DotNetBuildSourceOnly)' == 'true'` part sounds good to me.